en/docs/deploy-operate/observe/new-relic-integration.md

```
@@ -1,140 +1,163 @@
-    attributes:
-      service: order-service
-      environment: production
+
+## Step 4 - Send requests
+
+Run the service and send requests.
+
+Example cURL commands:
+
+```bash
+$ curl -X GET http://localhost:8090/shop/products
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)